assets/texts/braixen.txt

```diff
@@ -0,0 +1,3 @@
+Holy fricking crap. I want to hug the fox pokémon so goshdang bad. I can't stand it anymore. Every time I bring her into battle I want to hug her. I've seen literally every SFW fan art post there is of her online. My dreams are nothing but constant cuddling with Braixen. I'm sick of waking up every morning with my pillow in my arms and knowing that I should’ve spent that night hugging Braixen’s soft fox body. I want to feel her warm fur.
```

Comment:
You promised you were done! Also the vibes in this one are a little weird, even for me.